Added Code Sample and manual for S3 Object DataNode (#785)
Avaiga/taipy-doc: docs/manuals/core/config/data-node-config.md

does not need to spend much time configuring the *storage types* or the
*query system*. A predefined data node will often satisfy the user's
required format: pickle, CSV, SQL table, MongoDB collection, Excel sheet,

> Amazon Web Service S3 Object etc.

Reviewer: ```suggestion
Amazon Web Service S3 Object, etc.
```

    `Config.configure_in_memory_data_node()^` or the method `Config.configure_data_node()^`
    with parameter `storage_type="in_memory"`.

## Amazon Web Service S3 Object
A `S3ObjectDataNode^` is a specific data node used to model data stored in a
S3 bucket. To add a new *S3Object* data node configuration, the
`Config.configure_s3_object_data_node()^` method can be used. In addition to
the generic parameters described in the [Data node configuration](data-node-config.md)
section, multiple parameters can be provided.

> - _**aws_access_key**_ represents the Amazon Web Services identity account.

Reviewer: ```suggestion
- _**aws_access_key**_ represents the Amazon Web Services (AWS) identity account.
```

The abbreviation should be defined at the first appearance.

> - _**aws_secret_access_key**_ represents the Amazon Web Services access key to authenticate programmatic requests.

Reviewer: ```suggestion
- _**aws_secret_access_key**_ represents the AWS access key to authenticate programmatic requests.
```

> - _**aws_region**_ represnets the geographic area where Amazon Web Services (AWS) infrastructure is located.

Reviewer: ```suggestion
- _**aws_region**_ represnets the geographic area where the AWS infrastructure is located.
```

> - _**aws_s3_bucket_name**_ represnts the name aof the Amazon Web Service S3 bucket.

Reviewer: ```suggestion
- _**aws_s3_bucket_name**_ represnts the name of the AWS S3 bucket.
```

> - _**aws_s3_object_key**_ represents the name of the object(file) that has to be read or written.

Reviewer: ```suggestion
- _**aws_s3_object_key**_ represents the name of the object (file) that needs to be read or written.
```

> - _**aws _s3_object_parameters**_  represents additional arguments to be passed to interact with the AWS service.

Reviewer: ```suggestion
- _**aws _s3_object_parameters**_ represents additional arguments to be passed to interact with the AWS service.
```

> In this example, we configure an *s3_object* data node with the id "my_s3_objec":

Reviewer: ```suggestion
In this example, we configure a *S3Object* data node with the id "my_s3_object":
```
For consistency

- Its scope is the default value `SCENARIO`.

> - The object_key name is "taipy_object"

Reviewer 2: ```suggestion
- The object_key name is "taipy_object".
```

> - The object_key name  is "taipy_object"

Reviewer: ```suggestion
- The object_key name is "taipy_object"
```
